### PR TITLE
Document how to use Mise with Xcode projects and Xcode Cloud

### PR DESCRIPTION
**Pull request by pepicrft**
I'm documenting how to use Mise with Xcode projects and Xcode Cloud

### CODE REVIEW

File: docs/ide-integration.md [hunk 1]
```diff
@@ -28,6 +28,33 @@ Alternatively, you may be able to get tighter integration with a direnv extensio
...
+Xcode projects can system commands from script build phases and schemes. Since Xcode doesn't source your shell profile the tool versions pinned by your project's Mise configuration file are not automatically resolved through the `$PATH` environment variable.
+To work around this, you can use the `mise x`:
+
+```bash
+mise x -C $SRCROOT swiftlint
+```
+
+Note that the `-C` flag ensures that Mise will use the project's root directory, where the Mise configuration file lives, as the working directory for the command. Note that in schemes pre and post action scripts, you'll have to enable "Provide build settings from" to inherit the `$SRCROOT` environment variable.
+
+### Xcode Cloud
```
Comment (pepicrft) [inline docs/ide-integration.md]: @jdx what do you think if we have a page dedicated to CI providers? (e.g. GitHub Actions, CircleCI...)

Comment (jdx) [reply docs/ide-integration.md]: Yeah I think that'd be better than this page

Comment (pepicrft) [reply docs/ide-integration.md]: Extracted. Let me know what you think.

**Comment by jdx** [comment]
lgtm!

**Pull request merged**